src/components/pages/ManagementRegime/ManagementRegime.js

```diff
@@ -406,6 +409,7 @@ const ManagementRegime = ({ isNewManagementRegime }) => {
   )
 
   const deleteRecord = () => {
+    // only available online
```

Comment:
simplifying some of the logic that existed for a feature that doesnt exist (offline deleting)